programs/drift/src/controller/orders.rs

```diff
@@ -120,6 +122,16 @@ pub fn place_perp_order(
 
     validate!(!user.is_bankrupt(), ErrorCode::UserBankrupt)?;
 
+    if params.is_update_high_leverage_mode() {
+        if let Some(config) = high_leverage_mode_config {
+            let mut config = load_mut!(config)?;
+            config.update_user(user)?;
```

Comment:
perhaps its better not to fail here if unable to update to HLM, optimistically allow order to see if it passes margin check later on